src/components/SecuredRoute/SecuredRoute.tsx

```diff
@@ -13,4 +13,24 @@ const SecuredRoute = (props: any): JSX.Element => {
   );
 };
 
+let lastActive: number = Date.now();
+
+document.addEventListener('mousemove', () => {
+  lastActive = Date.now();
+});
+
+setInterval(() => {
+  const currentTime = Date.now();
+  const timeSinceLastActive = currentTime - lastActive;
+  const logoutUserTime = 20 * 60 * 1000;
+
+  if (timeSinceLastActive > logoutUserTime) {
+    toast.error(
+      'Session timed out. Please login again to keep using the platform'
+    );
+    window.location.href = '/';
+    localStorage.setItem('IsLoggedIn', 'FALSE');
```

Comment:
Use the already existing toast.

Reply:
So to clarify, are you saying I should use the existing toast but leave the inactive timeout function?

Reply:
don't leave the function, just use the existing toast.